Applications/VpView/vpViewCore.cxx

```diff
@@ -3236,15 +3243,11 @@ vpProject* vpViewCore::processProject(QScopedPointer<vpProject>& project)
       this->ViewExtents[2] = this->WholeImageBounds[2];
       this->ViewExtents[3] = this->WholeImageBounds[3];
 
-      if (project->AnalysisDimensions[0] == -1)
+      if (!project->AnalysisDimensions.isValid())
         {
-        project->AnalysisDimensions[0] =
-          this->ViewExtents[1] - this->ViewExtents[0] + 1;
-        }
-      if (project->AnalysisDimensions[1] == -1)
-        {
-        project->AnalysisDimensions[1] =
-          this->ViewExtents[3] - this->ViewExtents[2] + 1;
+        project->AnalysisDimensions = {
```

Comment:
Previous code set AnalysisDimensions width and height separately, test each individually  for valid or not.  Was that unnecessary?  I can't think of why that was the case - but did you verify it shouldn't be an issue?

Reply:
I guess not, but what would be the use case for that?

Reply:
Not sure - code dates back to 2012, so we're unlikely to figure that out.   I'm not too worried about it, but it caught my eye.